_posts/2021-01-25-troubleshooting-puppetized-dsc-modules.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: post
+title: "Troubleshooting Puppetized DSC Modules"
+author: michaeltlombardi
+---
+
+Occasionally, something will go wrong when trying to apply a manifest containing Puppetized DSC resources.
```

Comment:
This hook ought to be just a little longer to capture audience attention. Maybe if you just added "causing Puppet runs to fail" to the end it would be enough.